library/src/main/java/com/getstream/sdk/chat/rest/core/Client.java

```diff
@@ -825,8 +826,9 @@ public void updateChannel(@NonNull Channel channel, @Nullable Message updateMess
         onSetUserCompleted(new ClientConnectionCallback() {
             @Override
             public void onSuccess(User user) {
-                apiService.updateChannel(channel.getType(), channel.getId(), apiKey, clientID,
-                        new UpdateChannelRequest(channel.getExtraData(), updateMessage))
+                UpdateChannelRequest request = new UpdateChannelRequest(channel.getExtraData(), updateMessage);
+                Log.d(TAG, "Request: " + new Gson().toJson(request));
```

Comment:
Creating `Gson` instance is too much for logging purposes. It's compiled into release SDK code and just pure wasting of memory resource.